saylua/modules/forums/models/db.py

```diff
@@ -16,9 +38,12 @@ class Board(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     title = db.Column(db.String(256))
-    url_slug = db.Column(db.String(256))
+    text_id = db.Column(db.String(256), unique=True)
```

Comment:
text_id is a weird name, m8y, there's an implication there that isn't necessarily true 
(Which is that text_id's are permanent, non-mutable identifiers)